### PR TITLE
feat(view): add ctrTranslate and fix typo

### PR DESCRIPTION
**Pull request by lrangelc**

### CODE REVIEW

File: apps/guest/src/home/pages/special-request/special-request.component.ts [hunk 1]
```diff
@@ -30,7 +29,6 @@ export class SpecialRequestComponent implements OnInit, OnDestroy {
   constructor(
     private requestService: RequestService,
     private sanitizer: DomSanitizer,
-    private zoneService: ZoneService,
```
Comment (ruedagato) [inline apps/guest/src/home/pages/special-request/special-request.component.ts:33]: why do you remove this service ?

File: libs/core/src/lib/components/zone-list/zone-list.component.html [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
-
   <ng-container *ngFor="let zone of $zones | async">
     <mat-checkbox
       (change)="select($event, zone)"
       [checked]="zonesSelected[zone.uid]"
       color="primary"
       class="contler-zone__check"
     >
-      {{ zone.name }}
+      {{ zone.name | ctrTranslate | async }}
```
Comment (ruedagato) [inline libs/core/src/lib/components/zone-list/zone-list.component.html:20]:
```suggestion
      {{ zone.name | ctrTranslate }}
```

File: libs/core/src/lib/core.module.ts [hunk 3]
```diff
@@ -62,6 +63,7 @@ const materialModules = [
     FormsModule,
     AngularFireDatabaseModule,
     HttpClientModule,
+    UiModule,
```
Comment (ruedagato) [inline libs/core/src/lib/core.module.ts:66]: change it fros DynamicTranslateModule

**Review by ruedagato — CHANGES_REQUESTED** [review]

File: apps/guest/src/home/pages/special-request/special-request.component.ts (hunk 1 above)
Comment (lrangelc) [reply apps/guest/src/home/pages/special-request/special-request.component.ts:33]: it was not being used

**Review by ruedagato — APPROVED** [review]

**Pull request merged**